src/datachain/lib/dc.py

```diff
@@ -1499,22 +1500,41 @@ def from_records(
         Parameters:
             to_insert : records (or a single record) to insert. Each record is
                         a dictionary of signals and theirs values.
+            schema : describes chain signals and their corresponding types
 
         Example:
             ```py
-            empty = DataChain.from_records()
             single_record = DataChain.from_records(DataChain.DEFAULT_FILE_RECORD)
```

Comment:
okay, but is the meaning, purpose of this?

Reply:
You are referring to that example of creating dataset from single record or?

Reply:
yep, I would say this specific record - DEFAULT_FILE_RECORD - what is the point of this example?